torch/_dynamo/variables/tensor.py

```diff
@@ -1136,6 +1136,20 @@ def create(cls, tx, proxy, sym_num=None, **options):
             sym_num = int(sym_num) if isinstance(sym_num, sympy.Integer) else sym_num
             return ConstantVariable.create(sym_num)
 
+        def _is_unbacked_symbol(s):
+            from torch.fx.experimental.symbolic_shapes import (
+                free_unbacked_symbols,
+                is_symbolic,
+            )
+
+            return (
+                is_symbolic(s)
+                and isinstance(s.node.expr, sympy.Symbol)
+                and len(free_unbacked_symbols(s.node.expr)) > 0
```

Comment:
Can this ever be > 1?

Reply:
Also, are you just looking for is_unbacked_symint ?

Reply:
yeah, is_unbacked_symint is more accurate. But let me add more tests that exersize this code.